include/swift/AST/Stmt.h

```diff
@@ -808,25 +809,28 @@ class ForEachStmt : public LabeledStmt {
 public:
   ForEachStmt(LabeledStmtInfo LabelInfo, SourceLoc ForLoc, Pattern *Pat,
               SourceLoc InLoc, Expr *Sequence, Expr *WhereExpr, BraceStmt *Body,
-              Optional<bool> implicit = None)
+              bool isOptional, Optional<bool> implicit = None)
     : LabeledStmt(StmtKind::ForEach, getDefaultImplicitFlag(implicit, ForLoc),
                   LabelInfo),
       ForLoc(ForLoc), Pat(nullptr), InLoc(InLoc), Sequence(Sequence),
-      WhereExpr(WhereExpr), Body(Body) {
+      WhereExpr(WhereExpr), Body(Body), isOptional(isOptional) {
     setPattern(Pat);
   }
-  
+
```

Comment:
Please remove the whitespace-only diffs like this one (and several others).

Reply:
Those are from the last gardening commit (extra whitespaces I noticed *while* working on the proposal), I intendedly separated them. You want me to remove the commit?

Reply:
Yes, you could replace it with just the typo fixes.